File: PROGRAMMERS/LEVEL1/문자열 내림차순으로 배치하기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(string s) {
    string answer = "";

    //sort(s.begin(),s.end(),greater<char>()); 대치가능
    sort(s.begin(), s.end());
    reverse(s.begin(), s.end());
    answer = s;
    return answer;
}

File: PROGRAMMERS/LEVEL1/문자열 내림차순으로 배치하기.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.*;

class Solution {
    public String solution(String s) {
        
    	char[] str = s.toCharArray();
    	Arrays.sort(str);
    	
    	StringBuilder sb = new StringBuilder(new String(str));
    	
    	
    	return sb.reverse().toString();
    }
}